pkg/ingester/instance.go

```diff
@@ -8,6 +8,7 @@ import (
 	"syscall"
 
 	"github.com/cortexproject/cortex/pkg/cortexpb"
+	cutil "github.com/cortexproject/cortex/pkg/util"
```

Comment:
How come this is kept?

Reply:
@jeschkies the `github.com/cortexproject/cortex/pkg/util` package isn't part of the PR's topic though? Only ``github.com/cortexproject/cortex/pkg/util/log`.